includes/data-stores/class-wc-order-data-store-cpt.php

```diff
@@ -76,7 +76,6 @@ class WC_Order_Data_Store_CPT extends Abstract_WC_Order_Data_Store_CPT implement
 	public function create( &$order ) {
 		$order->set_order_key( 'wc_' . apply_filters( 'woocommerce_generate_order_key', uniqid( 'order_' ) ) );
 		parent::create( $order );
-		do_action( 'woocommerce_new_order', $order->get_id() );
```

Comment:
If you create an order through the admin or the API the `woocommerce_new_order` action won't get triggered any more. That seems like it could cause some problems.